pagopa/io-app
# feat: [IOCOM-570] Remove `newPnMessageDetailsEnabled` feature flag

## Short description
This PR removes the `newPnMessageDetailsEnabled` feature flag and all the related references.

## List of changes proposed in this pull request
- Remove local FF
- Remove legacy components

## How to test
Using `io-dev-api-server`, navigate to the detail of a PN message and check that everything is working properly.
